packages/language-server/src/utils/files/resolveTemplate.ts

```diff
@@ -0,0 +1,41 @@
+import { fileStat } from './fileStat';
+
+// HACK: Maybe we should move these to a configuration setting?
+const extensions = [`.twig`, `.html`];
```

Comment:
I don't see a reason why we should.

What if we replace this line with a link to the Craft source fragment, where these extensions are handled? That's hardcoded there, and thus should be hardcoded here, I guess...

Reply:
> Twig itself [doesn't enforce](https://twig.symfony.com/doc/3.x/api.html#rendering-templates) a `.twig` extension

As far as I understand, it does not require the included file to have `.twig` extension, although it requires the included file to exist, strictly by the provided path. So you can `include('somefile.txt')`, if it exists at `templates/somefile.txt`.